docs/documentation/quartz-3.x/how-tos/reschedule-jobs.md

```diff
@@ -0,0 +1,49 @@
+# Reschedule Jobs
+
+A few ways to approach a need to reschudel a job.
+
+## Manually Retry
+
+When a Quartz job is running, and an unhandled exception escapes the `IJob`, the Quartz system will mark the job in an error state. This would then allow you to reschedule the job using any method that would be work for your system. 
+
+## Polly Retries
+
+If your job simply needs to retry its work, then you could wrap the job in a [Polly](https://github.com/App-vNext/Polly) policy, and use the policy definitions to retry it.
+
+## Self-Rescheduling
+
+If your job needs more time, say it needs to wait 5 minutes, the `IJobExecutionContext` has access to the scheduler on it. You could use that to reschudel the job, and let it exit normally.
```

Comment:
job could also schedule also another job, this is a way to get job-chaining / workflows: https://www.quartz-scheduler.net/documentation/faq.html#how-do-i-chain-job-execution-or-how-do-i-create-a-workflow

Reply:
I think this becomes another how to.